src/sentry/api/endpoints/auth_index.py

```diff
@@ -221,7 +221,7 @@ def put(self, request: Request) -> Response:
 
         if not (request.user.is_superuser and request.data.get("isSuperuserModal")):
             if not validator.is_valid():
-                return self.respond(validator.errors, status=status.HTTP_400_BAD_REQUEST)
+                raise NotAuthenticated(detail=MISSING_AUTH_ERROR_MESSAGE)
```

Comment:
usually when the serializer is invalid we return serializer.errors (which is `validator.errors` in this case)

Reply:
same thing as above